.travis.yml

```diff
@@ -5,9 +5,9 @@ python:
 script:
   - # @TODO run tests here
   - pip install .
-  - "nosetests && pylint pommerman"
-  - "nosetests && pylint mananger"
-  - "nosetests && pylint examples"
+  - "nosetests && pylint -j 0 pommerman"
+  - "nosetests && pylint -j 0 mananger"
+  - "nosetests && pylint -j 0 examples"
```

Comment:
So this is just running on 1 thread. Will this really make this faster?

Reply:
The `-j 0` parameter auto-detects the amount of cores and it to that.

Reply:
Is that ... desirable? It means that suddenly all of your cores are going to this without being notified. OTOH, it is a really lightweight task.